### PR TITLE
BPR-3153: Added documentation for new REST Endpoints of StructuredPdf…

### PR DESCRIPTION
**Pull request by IgorDud-CDI**
… feature

### CODE REVIEW

File: rest-doc/examples/EmailSending.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "subject": "A hitchhiker's guide",
+  "date": "2018-02-28T13:37:42.000Z",
+  "text": "A long time ago, in a galaxy far far away.",
+  "html": "<html><body><p>A long time ago, in a galaxy far far away.</body></html>",
+  "attachments": [{
+    "contentType": "image/png",
+    "filename": "my-unicorn.png",
+    "content": "..."
+  }]
+}
```
Comment (nczeranka) [inline rest-doc/examples/EmailSending.json]: Not visible in generated md

File: rest-doc/examples/StructuredPdfChannelConfiguration.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "config": {
+    "supplierId": "hard001",
+    "escalationEmail": "scott.tiger@example.com",
+    "changedOn": "2020-02-26T14:56:25.000Z",
+    "changedBy": "scott.tiger@example.com",
+    "createdOn": "2020-02-26T14:56:25.000Z",
+    "createdBy": "scott.tiger@example.com",
+    "emails": ["scott.tiger@example.com"],
+    "email": "scott.tiger@example.com"
```
Comment (nczeranka) [inline rest-doc/examples/StructuredPdfChannelConfiguration.json]: That's not correct:
{
  "config": {
    "supplierId": "AAA",
    "escalationEmail": "b@b.de",
    "status": "activated",
    "createdBy": "a@a.de",
    "createdOn": "2020-02-27T10:02:59.000Z",
    "changedBy": "ocadmin",
    "changedOn": "2020-02-27T10:03:02.000Z"
  },
  "emails": [
    {
      "id": 61,
      "supplierId": "AAA",
      "email": "a@a.de"
    }
  ]
}

File: rest-doc/examples/EmailSending.json (hunk 1 above)
Comment (IgorDud-CDI) [reply rest-doc/examples/EmailSending.json]: Schema was missing to this example, I added it now

File: rest-doc/examples/StructuredPdfChannelConfiguration.json (hunk 2 above)
Comment (IgorDud-CDI) [reply rest-doc/examples/StructuredPdfChannelConfiguration.json]: I changed this and created some Insomnia Requests to check einvoice-send/structured-pdf, I also shared these Requests with you through Slack, so check them out if you got time :)

**Pull request merged**